src/tweenable.js

```diff
@@ -582,6 +587,16 @@ export class Tweenable {
   catch(onRejected) {
     return this.then().catch(onRejected)
   }
+  /**
+   * @method Tweenable#finally
+   * @param {function} onFinally Receives {@link shifty.promisedData} as the
+   * first parameter.
```

Comment:
Based on the test and the `@return` below, it seems that `onFinally`'s first parameter is actually `undefined`. Does this description need to be updated?